src/components/FindTalentsBtn/FindTalentsBtn.tsx

```diff
@@ -0,0 +1,20 @@
+'use client'
+import React from 'react'
+import { Button } from '@/components/Button/Button'
+import { AppRoutes } from '@/utils/routes'
+import { useRouter } from 'next/navigation'
+
+const FindTalentsBtn = () => {
+  const router = useRouter()
+
```

Comment:
**nitpick:** Let's return null if current route is `AppRoute.profiles`, so it does not show when we are currently at this route